astropy/units/core.py

```diff
@@ -2110,6 +2117,10 @@ def is_unity(self):
         unit = self.decompose()
         return len(unit.bases) == 0 and unit.scale == 1.0
 
+    @property
+    def first_base(self):
+        first_bases = [b.first_base for b in self.bases]
+        return CompositeUnit(self.scale, first_bases, self.powers)
```

Comment:
The original context to this issue was #3835, where I needed to be able to check if a prefixed unit's base unit was already included in a set of units.  As @mdboom wrote in #4037, stripping the prefixes off the base units of a composite unit is not as meaningful.

Maybe this is just a narrow use case after all, but I was still surprised that there wasn't a more obvious way to get at this info.  Maybe it would be simpler and cleaner if instead of special-casing "I need the base unit of a prefixed unit" we just expose the `._represents` attribute publicly as a read-only property?

Then, for example, `GeV.represents.bases[0]` will get me what I need.  This requires some working knowledge of the fact that a Unit may be defined in terms of another unit (captured in `represents`) and that a unit may have some base units (including when a unit just has a scale).